triagebot.toml

```diff
@@ -691,6 +679,41 @@ message_on_remove = "Issue #{number}'s nomination has been removed. Thanks all f
 message_on_close = "Issue #{number} has been closed. Thanks for participating!"
 message_on_reopen = "Issue #{number} has been reopened. Pinging @*T-types*."
 
+[notify-zulip."beta-nominated".compiler]
+required_labels = ["T-compiler"]
+zulip_stream = 474880 # #t-compiler/backports
+topic = "#{number}: beta-nominated"
+message_on_add = [
+    """\
+@**channel** PR #{number} "{title}" has been nominated for beta backport.
+""",
+    """\
+/poll Approve beta backport of #{number}?
+approve
+decline
+don't know
```

Comment:
Discussion: there's also "revisit next week" that I find we use fairly often. Should we also include that?

Reply:
Fair enough

Reply:
Right, I kinda was still thinking of the manual version